examples/mdetr/test/test_contrastive_alignment_loss.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+from examples.mdetr.contrastive_alignment_loss import contrastive_alignment_loss
+from test.test_utils import assert_expected, set_rng_seed
+from transformers import RobertaTokenizerFast
+
+
+@pytest.fixture(scope="class", autouse=True)
```

Comment:
we decided on using function scope as default right? but if this changes all the output values then let's leave it